packages/fxa-auth-server/test/local/routes/totp.js

```diff
@@ -741,6 +741,115 @@ describe('totp', () => {
       assert.calledOnceWithExactly(db.consumeRecoveryCode, 'uid', '1234567890');
     });
   });
+
+  describe('/totp/replace/start', () => {
+    it('should create a new TOTP token if user has an existing token', async () => {
```

Comment:
Do you want a test case for an unverified session? That's the only other branching logic I see not being covered.